graphql/schema/types/image.graphql

```diff
@@ -9,4 +9,5 @@ type Image {
   lastDeployed: Time!
   name: String!
   versionId: String!
+  distros: [Distro!]!
```

Comment:
Let's add an `id` field to this object, and rearrange the fields to be in alphabetical order